### ISSUE TITLE
JsonParseException on gitbucket home screen

### ISSUE THREAD

**Issue by nonamedguy** [gitbucket/gitbucket#2591]
Getting the following error on home screen, it seems to be related to some commit message, all the commits in the repo were made from cmd or atom.io editor and viewing them inside the editor or cmd is working fine so it seems to be a problem only on gitbuckets home screen, also accessing the repo directly by its url "host/root/dpsUI/" works fine in gitbucket and even viewing the commit messages works.

This is the error msg:
com.fasterxml.jackson.core.JsonParseException: Illegal character ((CTRL-CHAR, code 0)): only regular white space (\r, \n, \t) is allowed between tokens at [Source: (String)"{"userName":"root","repositoryName":"dpsUI","activityUserName":"root","activityType":"push","message":"[user:root] pushed to [branch:root/dpsUI#master] "[truncated 291 chars]; line: 1, column: 2]

![gb_home_error](https://user-images.githubusercontent.com/2465335/97735242-5a977480-1ada-11eb-8388-556da2505795.png)

**Comment by takezoe** [comment]
Hmm... Commit message might contain illegal characters as JSON? The activity log is stored in `~/.gitbucket/activity.log` by default. Do you see something wrong in this file? 

Or if you just want to recover your GitBucket instance, deleting this file can clear all activity logs.

**Comment by pbpf** [comment]
> Hmm... Commit message might contain illegal characters as JSON? The activity log is stored in `~/.gitbucket/activity.log` by default. Do you see something wrong in this file?
> 
> Or if you just want to recover your GitBucket instance from this error, deleting this file can clear all activity logs.

Yes, I have same issue, and I edit ~/.gitbucket/activity.log  according to your suggestion. It's fixed!
